symplyphysics/laws/conservation/mixture_mass_equal_sum_of_components_masses.py

```diff
@@ -0,0 +1,35 @@
+from typing import Sequence
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, print_expression, Symbol, validate_input,
+    validate_output)
+from symplyphysics.core.operations.sum_array import SumArray
+from symplyphysics.core.symbols.symbols import tuple_of_symbols
+
+# Description
+## The mass of a mixture of liquids (gases) is equal to the sum of the masses of the components of the mixture
+## sum(m_i) = m
+## Where:
+## m_i - mass of mixture component
+## m - mass of mixture
+##
+## NOTICE: This law can be applied to a mixture that is in a closed impenetrable volume,
```

Comment:
Use standard conditions format, like in other laws.
Also add condition that mass is not transformed to energy, for example due to annihilation.